### PR TITLE
Modify best practice

### PR DESCRIPTION
**Pull request by tennisonchan**
Modify best-practices in js, html and css.

### CODE REVIEW

File: best-practices/README.md [hunk 1]
```diff
@@ -166,6 +166,10 @@ JavaScript
 ----------
 
 * Use CoffeeScript.
+* Beware of closure as it can cause memory leak issue.
+* Avoid var self = this. OR var that = this OR ... Use $.proxy if needed
```
Comment (matteodepalo) [inline best-practices/README.md]: What about using the ES6 fat arrow?

File: best-practices/README.md [hunk 2]
```diff
@@ -166,6 +166,10 @@ JavaScript
 ----------
 
 * Use CoffeeScript.
```
Comment (matteodepalo) [inline best-practices/README.md]: This could also be removed :)

Comment (tennison35) [inline best-practices/README.md]: Good Idea :+1: :)

File: best-practices/README.md (hunk 1 above)
Comment (tennison35) [inline best-practices/README.md]: Yup! :)

**Comment by matteodepalo** [comment]
:+1:

**Pull request merged**